packages/fiori-generator-shared/package.json

```diff
@@ -27,7 +27,14 @@
         "!dist/**/*.map"
     ],
     "dependencies": {
-        "@sap-ux/project-access": "workspace:*"
+        "@sap-ux/project-access": "workspace:*",
+        "mem-fs": "2.1.0",
+        "mem-fs-editor": "9.4.0",
+        "i18next": "21.10.0"
```

Comment:
Is this `i18next` dependency needed?

Reply:
Thanks Cian for spotting this, I have removed this dependency.